client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -0,0 +1,360 @@
+				{
+					context: 'Notice for mapped domain notice with NS records pointing to somewhere else',
+				}
+			);
+			secondaryMessage = translate(
+				"Please note that it can take up to 72 hours for your changes to become available. If you're still not seeing your site loading at %(domainName)s, please wait a few more hours, clear your browser cache, and try again. {{learnMoreLink}}Learn all about mapping existing domain in our support docs{{/learnMoreLink}}.",
+				{
+					components: { learnMoreLink: learnMoreLink( MAP_EXISTING_DOMAIN ) },
+					args: { domainName: domain.name },
+				}
+			);
+		}
+
+		return (
+			<React.Fragment>
+				<div>
+					<p>{ primaryMessage }</p>
+					{ ! isSubdomain( domain.name ) && (
+						<ul>
+							{ WPCOM_DEFAULTS.map( nameServer => {
+								return <li key={ nameServer }>{ nameServer }</li>;
+							} ) }
+						</ul>
+					) }
+				</div>
+				<div>{ secondaryMessage }</div>
+			</React.Fragment>
+		);
+	}
+
+	renderDefaultRenewButton() {
+		const { domain, translate } = this.props;
+
+		if ( domain.expired || isExpiringSoon( domain, 30 ) ) {
+			return null;
+		}
+
+		if ( domain.bundledPlanSubscriptionId ) {
+			return (
+				<div>
+					<RenewButton
+						compact={ true }
+						selectedSite={ this.props.selectedSite }
+						subscriptionId={ parseInt( domain.bundledPlanSubscriptionId, 10 ) }
+						customLabel={ translate( 'Renew your plan' ) }
+						tracksProps={ { source: 'mapped-domain-status', mapping_status: 'active-plan' } }
+					/>
+				</div>
+			);
+		}
+
+		return (
+			<div>
+				<RenewButton
+					compact={ true }
+					selectedSite={ this.props.selectedSite }
+					subscriptionId={ parseInt( domain.subscriptionId, 10 ) }
+					tracksProps={ { source: 'mapped-domain-status', mapping_status: 'active' } }
+				/>
+			</div>
+		);
+	}
+
+	renderAutoRenew() {
+		return (
+			<Card compact={ true }>
+				Auto Renew (on) <CompactFormToggle checked={ true } />
+			</Card>
+		);
+	}
+
+	handlePaymentSettingsClick = () => {
+		this.props.recordPaymentSettingsClick( this.props.domain );
+	};
+
+	render() {
+		const { domain, moment, translate } = this.props;
+		const { name: domain_name } = domain;
+
+		const { statusText, statusClass, icon } = this.resolveStatus();
+
+		const newStatusDesignAutoRenew = config.isEnabled( 'domains/new-status-design/auto-renew' );
+
+		return (
+			<div className="domain-types__container">
+				<DomainStatus
+					header={ domain_name }
+					statusText={ statusText }
+					statusClass={ statusClass }
+					icon={ icon }
+				>
+					{ this.renderSettingUpNameservers() }
+					{ this.renderExpiringSoon() }
+				</DomainStatus>
+				<Card compact={ true } className="domain-types__expiration-row">
+					<div>
+						{ domain.bundledPlanSubscriptionId
+							? translate( 'Expires with your plan on %(expiry_date)s', {
+									args: {
+										expiry_date: moment( domain.expiry ).format( 'LL' ),
```

Comment:
Kudos on using `moment.utc( expiry )` in other parts of the code - like you yourself discovered, otherwise moment assumes the user's timezone. So let's make sure we do it here too (and some other places).